source/connect/tls.txt

```diff
@@ -0,0 +1,334 @@
+         :start-after: start-tls-mongo-client-settings
+         :end-before: end-tls-mongo-client-settings
+         :language: kotlin
+         :copyable:
+         :dedent:
+
+.. _tls_configure-certificates:
+
+Configure Certificates
+----------------------
+
+{+language+} applications that initiate TLS requests require access to
+cryptographic certificates that prove the application's identity and verify
+other applications with which the {+language+} application interacts. You can configure
+access to these certificates in your application in the following ways:
+
+- Using a JVM trust store and JVM key store
+- Using a client-specific trust store and key store
+
+.. _kotlin-sync-tls-configure-jvm-truststore:
+
+Configure the JVM Trust Store
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. note::
+
+   By default, the JRE includes many commonly used public certificates
+   from signing authorities such as `Let's Encrypt
+   <https://letsencrypt.org/>`__. As a result, you can connect to a
+   :atlas:`MongoDB Atlas </>` instance, or any other
+   server whose certificate is signed by an authority in the JRE's default
+   certificate store, with TLS enabled without configuring the trust store.
+
+The JVM trust store saves certificates that securely identify other
+applications with which your {+language+} application interacts. By using these
+certificates, your application can prove that the connection to another
+application is genuine and secure from tampering by third parties.
+
+If your MongoDB instance uses a certificate that is signed by an
+authority that is not present in the JRE's default certificate store,
+your application must configure the following system properties to initiate
+TLS requests.
+
+- ``javax.net.ssl.trustStore``: Path to a trust store containing the client's TLS
+  certificates
+
+- ``javax.net.ssl.trustStorePassword``: Password to access the trust
+  store defined in ``javax.net.ssl.trustStore``
+
+These properties ensure that your application can
+validate the TLS certificate presented by a connected MongoDB instance.
+
+You can create a trust store by using the `keytool <https://docs.oracle.com/javase/8/docs/technotes/tools/unix/keytool.html>`__
+command line tool from the JDK as shown in the following terminal command:
+
+.. code-block:: console
+
+   keytool -importcert -trustcacerts -file <path to certificate authority file>
+            -keystore <path to trust store> -storepass <password>
+
+Configure the JVM Key Store
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. note::
+
+   By default, MongoDB instances do not perform client certificate
+   validation. You must configure the key store if you configured your MongoDB
+   instance to validate client certificates.
+
+The JVM key store saves certificates that securely identify your {+language+}
+application to other applications. By using these certificates, other
+applications can prove that the connection to your application is
+genuine and secure from tampering by third parties.
```

Comment:
S: I think this is a duplicate paragraph from the prev section. Consider removing